data/olm-catalog/ssp-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,403 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
This file does not need to be commited to the repo. It will be generated by makefile when it is needed.